src/dmd/declaration.d

```diff
@@ -715,8 +715,8 @@ extern (C++) final class AliasDeclaration : Declaration
     extern (D) this(const ref Loc loc, Identifier ident, Type type)
     {
         super(loc, ident);
-        //printf("AliasDeclaration(id = '%s', type = %p)\n", id.toChars(), type);
-        //printf("type = '%s'\n", type.toChars());
+        // printf("AliasDeclaration(id = '%s', type = %p)\n", ident.toChars(), type);
+        // printf("type = '%s'\n", type.toChars());
```

Comment:
for a smaller diff, revert your changes in this module

Reply:
Done.